src/cli/cli.ts

```diff
@@ -59,7 +61,7 @@ function tokenByAddrOrSymbol(addressOrSymbol: string) {
 
 async function main() {
 	const opts = program.opts()
-	const kit = await newKit(opts.network)
```

Comment:
latest dependency bump brought in a version of contractKit that doesn't accept `wss://forno...` because it always assumes its a HTTP provider url.

Use `newKitFromWeb3` and use a real Web3 instance instead